### PR TITLE
2674: Create /download/core/intel-nuc page

### LINKED ISSUE #2674: Create the /download/core/intel-nuc page
create the www.ubuntu.com/download/core/intel-nuc page

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Created /download/core/intel-nuc page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/core/intel-nuc
- Check that content matches the [copydoc](https://docs.google.com/document/d/1e2aWvVEEXvxDWSbxDzs-_tqKbKbW9sapjCjIRiuAdMg/edit#)

## Issue / Card

Fixes #2674

### CODE REVIEW

File: templates/download/core/intel-nuc.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        <li class="p-list-step__item">
+          <h3 class="p-list-step__title">
+            <span class="p-list-step__bullet">4</span>
+            Flash Ubuntu Core
+          </h3>
+          <div class="p-list-step__content">
+            <ol class="u-no-margin--left">
+              <li>Once the system is ready, insert the second USB flash drive containing the Ubuntu Core image file.</li>
+              <li>Open a terminal and type the following command to check for directories mounted on the internal storage:
+                  <pre><code>mount | grep mmcblk</code></pre>
```
Comment (pmahnke) [inline templates/download/core/intel-nuc.html]: needs top margin

File: templates/download/core/intel-nuc.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+{% block title %}Install Ubuntu on the Intel NUC{% endblock %}}
+
+{% block content %}
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-10">
+      <div>
+        <h1>Install Ubuntu on the Intel NUC</h1>
+        <p>There are two install options for the Intel NUC: Ubuntu Core or Ubuntu Desktop.</p>
```
Comment (pmahnke) [inline templates/download/core/intel-nuc.html]: question... do you think these should be internal links?

File: templates/download/core/intel-nuc.html [hunk 3]
```diff
@@ -0,0 +1,213 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Install Ubuntu on the Intel NUC{% endblock %}}
```
Comment (pmahnke) [inline templates/download/core/intel-nuc.html]: does the Intel need a &lt;&reg;>?

File: templates/download/core/intel-nuc.html [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+    <div class="col-12 p-card">
+      <div class="u-equal-height p-divider">
+        <div class="col-6 p-divider__block">
+          <h2>Install Ubuntu Core</h2>
+          <p>We will walk you through the steps of flashing Ubuntu Core on an Intel NUC. At the end of this process, you will have a board ready for production or testing snaps.</p>
+        </div>
+        <div class="col-6 p-divider__block">
+          <h3>Minimum requirements</h3>
+          <ul class="p-list">
+            <li class="p-list__item is-ticked">An Intel® NUC with BIOS updated to the latest version (<a class="p-link--external" href="http://www.intel.com/content/www/us/en/support/boards-and-kits/000005850.html">update instructions</a>)</li>
```
Comment (pmahnke) [inline templates/download/core/intel-nuc.html]: and remove the &amp;reg; from here

File: templates/download/core/intel-nuc.html (hunk 2 above)
Comment (Caleb-Ellis) [reply templates/download/core/intel-nuc.html]: Yeah I think that's a good idea

File: templates/download/core/intel-nuc.html (hunk 3 above)
Comment (Caleb-Ellis) [reply templates/download/core/intel-nuc.html]: I don't think you can do superscript on the title, but I can put the registered trademark symbol in

**Pull request merged**